integration_test/src/test_runner/mod.rs

```diff
@@ -27,11 +28,27 @@ pub enum Error {
         expected: usize,
         actual: usize,
     },
-    #[error("mismatch in column counts for test {test}: expected {expected}, actual {actual}")]
-    ColumnCount {
+    #[error("mismatch in results column counts for test {test}: expected {expected}, actual {actual}, row {row}")]
```

Comment:
```suggestion
    #[error("mismatch in result set column counts for test {test}: expected {expected}, actual {actual}, row {row}")]
```
Should this say "result" or "result set" (or "resultset") instead of "results"?

Reply:
Good catch, I'll update that.